x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,529 @@
+[[update-node-certs]]
+== Updating node security certificates
+If your cluster certificates expired or you have a new Certificate Authority
+(CA), then you'll need to update the security certificates for both the
+transport and HTTP layers. When updating your node security certificates, there
+are two main scenarios that you'll encounter:
+
+* <<update-node-certs-same,Your certificate authority (CA) or key remains the same>>
+* <<update-node-certs-different,Your CA certificate or key changes>>
+
+Regardless of the scenario, {es} monitors the SSL resources for updates
+on a five-second interval. You can just copy the new certificate and key files
+(or keystore) into the {es} configuration directory and your nodes will detect
+the changes. 
+
+Because {es} doesn't reload the `elasticsearch.yml` configuration,
+you must use *the same file names* so that {es} refreshes the security
+certificates and keys. This method of updating certificates is preferred
```

Comment:
I don;t think we need to make assumptions on what is the best for the users here? Furthermore, in most cases they can do this with rolling restarts (no service interruption)

Reply:
I think we can just delete this sentence beginning with "This method of updating..." and keep the preceding sentence.